rest-api/tools/import_organizations.py

```diff
@@ -181,13 +181,26 @@ def _entity_from_row(self, row):
                 link=link)
 
   def _get_lat_long_for_site(self, address_1, city, state):
+    full_address = address_1 + ' ' +  city + ' ' + state
     try:
         api_key = os.environ.get('API_KEY')
         gmaps = googlemaps.Client(key=api_key)
         geocode_result = gmaps.geocode(address_1 + '' +  city + ' ' +  state)[0]
-        latitude = geocode_result['geometry']['location']['lat']
-        longitude = geocode_result['geometry']['location']['lng']
-        return latitude, longitude
+        if geocode_result:
+          try:
+            latitude = geocode_result['geometry']['location']['lat']
+            longitude = geocode_result['geometry']['location']['lng']
+            if latitude != None and longitude != None:
+              return latitude, longitude
+            else:
+              logging.info('Can not find lat/long for %s', full_address) 
+              return None, None
+          except KeyError as e:
+            logging.warn('error for lat/long. There is no %s for this address', e)
```

Comment:
What is %s supposed to be substituted with here?

If you want to log the exception, you should probably use logging.exception I think...

Reply:
The substitution will be the key (lat/long).

Reply:
I don't see how that works... the arg here is "e" which is a KeyError, not a field name?

Reply:
It would say, "There is no lat for this address". Moot point now, as the key error checking has been removed.